plugins/sql_functions.cc

```diff
@@ -229,25 +249,39 @@ getStatus() const
 struct SqlExpressionFunctionApplier: public FunctionApplier {
     SqlExpressionFunctionApplier(SqlBindingScope & outerContext,
                                  const SqlExpressionFunction * function,
-                                 const SelectExpression & expression,
                                  const FunctionValues & input)
-        : FunctionApplier(function),
-          context(outerContext, input),
-          bound(expression.bind(context))
+        : FunctionApplier(function), function(function),
+          innerScope(const_cast<SqlExpressionMldbContext &>
```

Comment:
Probably a comment will do.  Since none of the methods of SqlExpressionMldbContext modify the object, it can't actually cause any harm.  I didn't want to create a new object for no reason, but maybe I should just do it and make it simple.